### DISCUSSION TITLE
Report and Forms reply

### DISCUSSION THREAD

**Discussion by ImpulsiveFPS** [gameserverapp/Platform#380] [Suggestions]
Sometime you might want to get some additional information from the user when they report something.
Would be a good idea if we could actually reply to the report and they get a message and/or discord DM.

The same thing could go to forms as sometimes people might have done a great for example whitelist application but they missed to fill in some crucial information that we need to be able to whitelist them fully.

Also the option to require Discord to be linked to be able to report as this would make it easier for us to be able to contact the user who sent in the report. 

An option that require the user to have the email registered could also be a good thing as else there is no way for the user to know if they got a reply.

**Comment by mvaessen** [comment]
You can set fields on forms as being `required`, which ensures people fill those in.
